app/components/navbar.hbs

```diff
@@ -0,0 +1,33 @@
+<nav class="nav">
+  <button type="button" class="hamburger--toggle" {{on "click" this.toggleNav}}>
+    <span class="hamburger__bar"></span>
+    <span class="hamburger__bar"></span>
+    <span class="hamburger__bar"></span>
+  </button>
+  <div class="user">
+    <div class="user__greet">
+      <div class="user__greet__msg">Hello, {{@firstName}}!</div>
+      <img class="user__profile__pic" src={{@profilePictureURL}} alt="profile"/>
+    </div>
+  </div>
+
+  <ul class="nav-menu {{if this.isNavOpen "active-nav" ""}}">
+    <li class="nav__menu__logo">
+      <a href={{this.HOME_PAGE_URL}}>
+        <img src="RDSLogo.png" alt="real_dev_squad" height="50px" width="50px" />
+      </a>
+    </li>
+    <li class="nav__menu__home"><a href={{this.HOME_PAGE_URL}}>Home</a></li>
+    <li><a href={{this.WELCOME_SITE_URL}}>Welcome</a></li>
+    <li><a href={{this.EVENTS_SITE_URL}}>Events</a></li>
+    <li><a href={{this.MEMBERS_SITE_URL}}>Members</a></li>
+    <li><a href={{this.CRYPTO_SITE_URL}}>Crypto</a></li>
+    <li><a href={{this.STATUS_SITE_URL}}>Status</a></li>

```

Comment:
repetition! I guess you already know what to do here!

Reply:
Yeah got it